fineract-provider/dependencies.gradle

```diff
@@ -34,6 +34,7 @@ dependencies {
     implementation(project(path: ':fineract-branch'))
     implementation(project(path: ':fineract-document'))
     implementation(project(path: ':fineract-progressive-loan'))
+    implementation(project(path: ':fineract-report'))
```

Comment:
Please list all the fineract-report entities in the persistence.xml of the fineract-provider

Reply:
fineract-report has not any entity here, It has just annotation, service provider and interfaces and service implementation